api/tests/conftest.py

```diff
@@ -47,7 +47,8 @@ def get_table_properties_from_template():
 
 def make_job(granule='S1B_IW_SLC__1SDV_20200604T082207_20200604T082234_021881_029874_5E38',
              description='someDescription',
-             job_type='RTC_GAMMA'):
+             job_type='RTC_GAMMA',
+             polygon=''):
```

Comment:
`polygon` isn't part of a job submission, I don't think this parameter is needed